src/main/java/frc/robot/end_effector/EndEffectorIOSim.java

```diff
@@ -0,0 +1,74 @@
+package frc.robot.end_effector;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.math.system.plant.LinearSystemId;
+import edu.wpi.first.wpilibj.simulation.DCMotorSim;
+
+public class EndEffectorIOSim implements EndEffectorIO {
+ 
+    private final DCMotorSim algaeIntake = new DCMotorSim(
+        LinearSystemId.createDCMotorSystem(DCMotor.getNeoVortex(1), 0.001, 1.0),
```

Comment:
You need to recalculate the moment of inertia for each of the motors from the CAD.